tests/orchestrations_test.go

```diff
@@ -207,6 +210,56 @@ func Test_SingleActivity(t *testing.T) {
 	)
 }
 
+func Test_SingleActivity_TaskSpan(t *testing.T) {
```

Comment:
There should be a test added in https://github.com/microsoft/durabletask-go/blob/main/tests/grpc/grpc_test.go as well to validate we are correctly passing the context information over grpc stream.